route_builder.go

```diff
@@ -206,10 +200,9 @@ func (b *RouteBuilder) Deprecate() *RouteBuilder {
 
 // ResponseError represents a response; not necessarily an error.
 type ResponseError struct {
-	Code      int
-	Message   string
-	Model     interface{}
-	IsDefault bool
```

Comment:
the field IsDefault cannot be removed as it breaks compatability.  Better add comment that states it is deprecated and one should use DefaultReturns method.

Reply:
I see. Added the field back with a comment.